examples/live/run.sh

```diff
@@ -195,10 +195,12 @@ python -m mpi4py `which pycbc_live` \
 --src-class-eff-to-lum-distance 0.74899 \
 --src-class-lum-distance-to-delta -0.51557 -0.32195 \
 --run-snr-optimization \
---snr-opt-di-maxiter 50 \
---snr-opt-di-popsize 100 \
---snr-opt-include-candidate \
---snr-opt-seed 42 \
+--snr-opt-extra-opts \
+    "--snr-opt-method differential_evolution \
+    --snr-opt-di-maxiter 50 \
+    --snr-opt-di-popsize 100 \
+    --snr-opt-seed 42 \
```

Comment:
can we remove this seed option?  Since we would like to check that the default seed works (ie it gives stable output)